### ISSUE TITLE
[FEATURE] Отображение состояния позиции

### ISSUE THREAD

**Issue by sbelashevskiy** [alor-broker/Astras-Trading-UI#712]
### Please describe, is your feature request related to a problem?

Сейчас состояние позиции можно частично отслеживать в нескольких виджетах:
1. Кол-во лотов в портфеле - в виджетах выставления заявки, скальперском стакане, в блоттере
2. Заявки по позиции - в блоттере (на разных вкладках), в стаканах (только лимитные)

Чтобы получить общее представление о состоянии позиции (текущее кол-во, активные заявки) придется смотреть в разных виджетах или же настраивать фильтры в на разных вкладках блоттера.

Могу предположить, что это не очень удобно пользователям, особенно при условии наличия по позиции нескольких заявок разного типа и с разными параметрами, которые могут изменить состояние позиции.
Высока вероятность, что, например, при закрытии позиции по рынку у пользователя остаются лимитные и стоп заявки, о которых он забыл.

### Describe the solution you would like

Создать виджет, который бы отображал полное текущее состояние конкретной позиции.

Под состоянием подразумеваю как минимум:
1. Текущее кол-во лотов (с цветовой индикацией long/short)
2. Активные заявки. Скорее просто флаги наличия (кол-во) активных лимитных и условных заявок (возможно, с цветовой индикацией buy/sell). Детализация по заявкам здесь, я думаю, не нужна т.к. ее можно уже посмотреть в блоттере. Эти элементы скорее просто напоминание о наличии заявок, которые могут изменить текущее состояние

Можно добавить какие-нибудь данные по объему и т.п.
В целом же кол-во отображаемой информаци должно быть минимально, чтобы виджет занимал как можно меньше места, чтобы его можно было разместить над/под стаканом, например. 

### Describe alternatives you've considered

Добавить на вкладку "Позиции"  блоттера столбец с отображение кол-ва активных лимитных и стоп заявок. 
Но это на крайний случай, т.к. в этом случае пользователю все равно придется настраивать фильтры в таблицах

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contribution guideline

**Comment by dzeron13** [comment]
Неистово плюсую. Сейчас следить за всеми своими заявками и позициями тяжело, приходится подглядывать на график в AlorTV. Вот, для примера:
![image](https://user-images.githubusercontent.com/99450543/205039976-ca3d8175-6162-41ce-b3f3-f10ce479f46c.png)
Когда-нибудь, надеюсь, и на графике TV в Astras тоже позиции и заявки появятся, но для тех, кто сидит на стандартном графике, указанный в таске функционал очень пригодится.

**Comment by PilgrimViis** [comment]
Да, давайте сделаем. А как это должно выглядеть? Может как-то так? 

![image](https://user-images.githubusercontent.com/6408195/210343217-fc7df922-71c9-4716-ae20-fa00b98e7ba3.png)

P.S. Понятное дело, что среднюю цену мы правильно не посчитаем, но было бы все равно полезно.
